packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -164,7 +176,7 @@ export const FeedContainer = ({
       )}
     >
       <ScrollToTopButton />
-      <div className="flex w-full flex-col pt-2 laptopL:mx-auto" style={style}>
+      <div className="flex w-full flex-col laptopL:mx-auto" style={style}>
```

Comment:
Is the `pt-2` removal intentional?

Reply:
Yes. We don't need that top padding here.